mobile/__tests__/MyICardView-test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import renderer, { act } from 'react-test-renderer';
+import MyICardView from '../src/views/MyICardView';
+
+
+
+test('refresh control is active or not', async () => {
+    const component = render (
+        <MyICardView/>
+    )
+
+    const scrollViewComp = component.getByTestId('refreshControl');
+    expect(scrollViewComp).toBeDefined();
+
+    const {refreshControl} = scrollViewComp.props;
+
+    await act(async () => {
+        refreshControl.props.onRefresh();
+    })
```

Comment:
I'm not sure I fully understand what this tests, if we are testing the on refresh should the call come before the expect?

Reply:
It was just a basic test to see how testing works on react native. Did tests for react before and wanted to see if it works the same way or not. Just a rudimentary test to see if onRefresh works or not I believe. But didn't get a chance to test it coz of all the jest issues that I had mentioned on the discord a while back :( Should I just remove it?

Reply:
Sure let's just leave them in, maybe add a comment explaining these tests were just an experiment.